src/Fields.php

```diff
@@ -428,23 +428,21 @@ protected function wf_crm_get_fields($var = 'fields') {
       $fields['activity_activity_date_time'] = array(
         'name' => t('Activity # Date'),
         'type' => 'date',
-        'value' => 'now',
+        'default_value' => 'now',
       );
       $fields['activity_activity_date_time_timepart'] = array(
         'name' => t('Activity # Time'),
-        'type' => 'webform_time',
-        'value' => 'now',
+        'type' => 'time',
+        'default_value' => 'now',
       );
       $fields['activity_duration'] = array(
         'name' => t('Activity # Duration'),
         'type' => 'number',
-        'extra' => array(
-          'field_suffix' => t('min.'),
-          'min' => 0,
-          'step' => 5,
-          'integer' => 1,
-        ),
-      );
+        'field_suffix' =>  t('min.'),
+        /*ToDo Figure out why setting min does not work!*/
+        'min' => 0,
+        'step' => 5,
+       );
       $tag_entities = array('other', 'activity');
```

Comment:
I pulled these out of the extras to get them working. We'll 'field_suffix' and 'step' work - but for some reason 'min' does not. Not sure why that is.